target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz3390.dts

```diff
@@ -149,12 +157,12 @@
 &gswip_mdio {
 	phy0: ethernet-phy@0 {
 		reg = <0x0>;
-		gpios = <&gpio 32 GPIO_ACTIVE_HIGH>;
```

Comment:
I would consider separating this fix into a separate PR. Perhaps this would make it easier to find someone to test this patch. If it turns out that it works then it should also be backported to 22.03.

Reply:
@abajk I have removed this and created a separate PR 10323 for this.